_examples/signals-to-json.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+/* Example usage, showing systemd unit property changes:
+ *
+ *   signals-to-json -systemBus=true -filter="type='signal',interface='org.freedesktop.DBus.Properties',member='PropertiesChanged',arg0='org.freedesktop.systemd1.Unit',path_namespace='/org/freedesktop/systemd1/unit'"
+ *
+ */
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"github.com/charles-dyfis-net/go-dbus"
+	"os"
+)
+
+var cfg struct {
+	Filter string
+	SystemBus bool
+}
+
+func main() {
+	var conn *dbus.Conn
+	var err error
+
+	flag.BoolVar(&cfg.SystemBus, "systemBus", false, "Use system rather than session bus")
+	flag.StringVar(&cfg.Filter, "filter", "", "Filter to apply to select signals (default looks for systemd service property changes)")
+	flag.Parse()
```

Comment:
I question the merits of having flags in an example... I think it dilutes the example, as it doesn't relate to its purpose. I suggest you simply use the session bus and use an example filter; it will make the whole thing more succinct.